ibis/backends/base/sql/alchemy/__init__.py

```diff
@@ -191,27 +197,25 @@ def create_table(
         )
 
         with self.begin() as bind:
-            t.create(bind=bind)
+            t.create(bind=bind, checkfirst=force)
```

Comment:
`not force`?

Reply:
I'm not following.

`checkfirst=True` means `CREATE TABLE IF NOT EXISTS` which is consistent with "force the creation of this table" (force is perhaps a misnomer here, since there's no destructive behavior)

Reply:
Force indicates "no matter what" for me.

Reply:
How is that different from what I described?

Reply:
`force` indicates destructive behavior for me: `DROP TABLE IF EXISTS...` with `CREATE TABLE...` in a transaction, though I assume we don't want that.

Reply:
`force` is a misnomer, but I think it's out of scope to change the name of this parameter in this PR. However, `force` is consistent with "don't fail if the thing already exists", which is what the current semantics are.